java/src/main/java/lesson03/README.md

```diff
@@ -188,27 +180,24 @@ Now change the `FormatterResource` handler method to use `startServerSpan`:
 ```java
 @GET
 public String format(@QueryParam("helloTo") String helloTo, @Context HttpHeaders httpHeaders) {
-    try (Scope scope = Tracing.startServerSpan(tracer, httpHeaders, "format")) {
+    try (Scope scope = startServerSpan(tracer, httpHeaders, "format")) {
         String helloStr = String.format("Hello, %s!", helloTo);
         scope.span().log(ImmutableMap.of("event", "string-format", "value", helloStr));
         return helloStr;
     }
 }
 ```
 
+#### Apply the same to the Publisher
+
```

Comment:
Later on, it is said that 5 spans are expected, but without explicitly saying that the publisher needs to be changed as well, the user might wonder why there are only 4 being reported.